pss/forwarding_test.go

```diff
@@ -324,7 +326,7 @@ func addPeers(kad *network.Kademlia, addresses []pot.Address) {
 }
 
 func createPss(t *testing.T, kad *network.Kademlia) *Pss {
-	privKey, err := crypto.GenerateKey()
+	privKey, err := utils.GenerateKey()
```

Comment:
If we have other `utils` than crypto later maybe we should leave name `crypto` in this symbol for clarity

Reply:
Sorry, these changes belong to #1731. This PR shouldn't be reviewed until that one is merged.